tests/std/tests/GH_003119_error_category_ctor/test.compile.pass.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <string>
+#include <system_error>
+
+using namespace std;
+
+#ifndef _M_CEE_PURE

```

Comment:
This is fine, but it would be slightly better to use the `impure_matrix.lst`, saving a couple of unnecessary compilations. (`impure_matrix.lst` is just `usual_matrix.lst` minus `/clr:pure`.)